### DISCUSSION TITLE
I have v6 on the redis-server, can I use v9 on the client?

### DISCUSSION THREAD

**Discussion by lingfengChen03** [redis/go-redis#2347] [Q&A] [answered]
See the title and expect someone to address my query. Thanks a lot

**Comment by lingfengChen03** [comment, marked as answer]
oh see https://github.com/go-redis/redis/discussions/2241

**Comment by monkey92t** [comment]
v9 already supports redis6-server

**Reply by lingfengChen03** [reply]
Thank you for your reply and Happy New Year
